Tools/cases_generator/formatting.py

```diff
@@ -58,13 +58,13 @@ def reset_lineno(self) -> None:
             self.set_lineno(self.lineno + 1, self.filename)
 
     @contextlib.contextmanager
-    def indent(self):
+    def indent(self) -> typing.Iterator[None]:
         self.prefix += "    "
         yield
         self.prefix = self.prefix[:-4]
 
     @contextlib.contextmanager
-    def block(self, head: str, tail: str = ""):
+    def block(self, head: str, tail: str = "") -> typing.Iterator[None]:
```

Comment:
It would be better to use `collections.abc.Iterator` here, since `typing.Iterator` is deprecated